src/gui/GraphicPacksWindow2.cpp

```diff
@@ -386,28 +388,29 @@ void GraphicPacksWindow2::LoadPresetSelections(const GraphicPackPtr& gp)
 		{
 			continue;
 		}
-					
-		wxString label(category.empty() ? _("Active preset") : wxString(category));
+
+		wxString categoryWxStr = wxHelper::FromUtf8(category);
+		wxString label(category.empty() ? _("Active preset") : wxString(categoryWxStr));
```

Comment:
Small nitpick but since `categoryWxStr` is already a wxString it shouldn't be necessary to wrap it in another `wxString(...)` constructor

Reply:
Good catch